docs/cluster-management/configure-api.md

```diff
@@ -161,19 +161,30 @@ datastore:
 
 ### Executor Plugin
 
-We currently support [kubernetes](https://github.com/screwdriver-cd/executor-k8s),  [docker](http://github.com/screwdriver-cd/executor-docker), [VMs in Kubernetes](https://github.com/screwdriver-cd/executor-k8s-vm), [nomad](http://github.com/lgfausak/executor-nomad), and [Jenkins](https://github.com/screwdriver-cd/executor-jenkins) executor. See the [custom-environment-variables file](https://github.com/screwdriver-cd/screwdriver/blob/master/config/custom-environment-variables.yaml) for more details.
+We currently support [kubernetes](https://github.com/screwdriver-cd/executor-k8s),  [docker](http://github.com/screwdriver-cd/executor-docker), [VMs in Kubernetes](https://github.com/screwdriver-cd/executor-k8s-vm), [nomad](http://github.com/lgfausak/executor-nomad), [Jenkins](https://github.com/screwdriver-cd/executor-jenkins), and [queue](https://github.com/screwdriver-cd/executor-queue) executors. See the [custom-environment-variables file](https://github.com/screwdriver-cd/screwdriver/blob/master/config/custom-environment-variables.yaml) for more details.
 
-#### Kubernetes
-Set these environment variables:
+#### Kubernetes (k8s)
+If you use this executor, builds will run in pods in Kubernetes.
 
 | Environment name   | Default Value | Description                                |
 |:-------------------|:--------------|:-------------------------------------------|
-| EXECUTOR_PLUGIN    | k8s           | Default executor (eg: `k8s`, `docker`, `k8s-vm`, `jenkins`) |
+| EXECUTOR_PLUGIN    | k8s           | Default executor (eg: `k8s`, `docker`, `k8s-vm`, `nomad`, `jenkins`, or `queue`) |
 | LAUNCH_VERSION     | stable        | Launcher version to use                    |
+| EXECUTOR_PREFIX    |               | Prefix to append to pod names              |
 | EXECUTOR_K8S_ENABLED | true        | Flag to enable Kubernetes executor         |
 | K8S_HOST           | kubernetes.default | Kubernetes host                       |
 | K8S_TOKEN          | Loaded from `/var/run/secrets/kubernetes.io/serviceaccount/token` by default | JWT for authenticating Kubernetes requests |
 | K8S_JOBS_NAMESPACE | default       | Jobs namespace for Kubernetes jobs URL     |
+| K8S_CPU_MICRO      | 0.5           | Number of CPU cores for micro              |
+| K8S_CPU_LOW        | 2             | Number of CPU cores for low                |
+| K8S_CPU_HIGH       | 6             | Number of CPU cores for high               |
+| K8S_MEMORY_MICRO   | 1             | Memory in GB for micro                     |
+| K8S_MEMORY_LOW     | 2             | Memory in GB for low                       |
+| K8S_MEMORY_HIGH    | 12            | Memory in GB for high                      |
+| K8S_BUILD_TIMEOUT  | 90            | Default build timeout for all builds in this cluster (in minutes) |
+| K8S_MAX_BUILD_TIMEOUT | 120        | Default max build timeout for all builds in this cluster (in minutes) |
```

Comment:
This sounds a little confusing because it sounds just like the line above. We want to say something like `users can only configure build timeout up to the value`. But idk how to put that either.

Reply:
How about:

"Maximum user-configurable build timeout for all builds in this cluster (in minutes)"?